app/src/main/kotlin/com/twain/interprep/presentation/ui/theme/Type.kt

```diff
@@ -12,6 +12,11 @@ val Typography = Typography(
         fontFamily = FontFamily.Default,
         fontWeight = FontWeight.Normal,
         fontSize = 16.sp
+    ),
+    bodyLarge = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.Bold,
+        fontSize = 20.sp
```

Comment:
Add fontSize to dimens.xml

Reply:
@Audienix when I try `fontSize = dimensionResource(id = R.dimen.dimension_16sp).value.sp` it gives the error:

```
@Composable invocations can only happen from the context of a @Composable function
```

Should I just leave it as `20.sp` then?